ginga/rv/plugins/Catalogs.py

```diff
@@ -112,6 +123,38 @@ def __init__(self, fv, fitsimage):
         self.hilite = None
         self.gui_up = False
 
+        bank = self.fv.get_ServerBank()
+
+        # add catalogs found in configuration file
+        catalogs = self.settings.get('catalog_sources', [])
+        for d in catalogs:
+            typ = d.get('type', None)
+            obj = None
+            if typ == 'astroquery.vo':
+                obj = catalog.AstroQueryVOCatalogServer(self.logger, d['fullname'],
```

Comment:
It is offically "Astroquery" and not "AstroQuery" according to their documentation.
```suggestion
                obj = catalog.AstroqueryVOCatalogServer(self.logger, d['fullname'],
```